### ISSUE TITLE
spotify_player stop music when quitting it

### ISSUE THREAD

**Issue by oliv3340** [aome510/spotify-player#567]
Hi, thanks for your work. Awesome app! I use it and love it a lot!

**Describe the bug**
When I launch a song in `spotify_player` (the Spotify app launches but does not play anything), it works but stops as soon as I quit the app from the terminal. However, if I launch a song from Spotify itself first, other manipulations work as expected. 
When I play music from `spotify_player`, I can relaunch it after quitting `spotify_player` using the "play" button on my keyboard, but it would be better if the music didn't stop at all. I don't know if something can be done about this.

**To Reproduce**
On macOS (I don't know about other systems):
1. Launch the Spotify app.
2. Play a song in `spotify_player`.
3. Quit `spotify_player` (the music should stop).

**Expected behaviour**
Music shouldn't stop after quitting `spotify_player`.

**Log and backtrace**
Only infos in logs (nothing intersting) and no backtrace on the issue.

**Environment**
- OS: MAC OS 15.0 (Apple silicon)
- Application version: spotify_player v0.19.1, spotify 1.2.47.364.gf06e5cee

**Additional context**
I've install spotify_player with homebrew.

**Comment by aome510** [comment]
hmm music stops when you close the app is kinda unexpected when streaming is handled by the app. Not sure what is the bug here. Do you specifically mean if you stream music from **an external source** and use `spotify_player` as the **remote control device**, quitting the app stops the music played the external source?

**Comment by derekbunch** [comment]
this also happens for me.

if i start streaming from the spotify app it will keep playing, but if i start playing from `spotify_player` it will stop as soon as spotify_player is quit.

if tested these scenarios
- spotify app is running, but not playing anything
- - start with spotify_player
- - quit spotify_player
- - music stops

- spotify app not running
- - start music with spotify_player
- - quit spotify_player
- - music stops

- start music with spotify app
- - change track with spotify_player
- - quit spotify_player
- - music still plays

- spotify app is running, but not playing anything
- - start music with spotify_player
- - change music with app
- - quit spotify_player
- - music stops

so it seems to be only when streaming originates from spotify_player

**Comment by aome510** [comment]
look expected to me. Not sure why you guys expect music to continue playing when exiting the `spotify_player` app

**Comment by derekbunch** [comment]
I don't think we expected it to keep running. It makes sense that it doesn't. 

I think we were just saying it would be nice if it could continue running after exiting (like with a daemon or something)
